kubernetes/README.md

```diff
@@ -0,0 +1,57 @@
+# Coordination Network Kubernetes config
+
+This is an example of how to run Coordination Network in Kubernetes.
+
+## Setting it up
+
+1. Copy the secrets template and replace the values:
+
+```bash
+cp kubernetes/secrets-template.yaml kubernetes/secrets.yaml
+```
+
+You can encode your secrets using:
+
+```bash
+echo -n 'your-secret-value' | base64
+```
+
+3. Apply the configs
+
+```bash
+kubectl apply -k .
+```
+
+4. Get a shell in the django pod
+
+```
+kubectl exec -it <DJANGO_POD_NAME> -n coordnet -- /bin/bash
+```
+
+And create an admin:
```

Comment:
We could move this somewhere more general, so that non-k8s users know about it and we only have to maintain it in one location.

Reply:
Where do you think is best for that? I can ada a section in the README or I could create a Wiki and start adding info there?

Reply:
The README seems best to me for now. Keeping it all in one place.